docs/en_US/guides/blocking-updates.md

```diff
@@ -10,20 +10,25 @@ extra_contributors:
 
 This will take you through installing the iOS 17 "tvOS profile" to block updates on your device. It's easy to install and ensures you'll stay on the firmware you're currently on. This is useful if you're waiting for a jailbreak for a lower version and don't want to update in case it gets patched.
 
+::: danger
+
+This will block updates until 31st January 2025. Hopefully this guide will have been updated by then!
+
+:::
+
 ## Installing the tvOS profile
 
 1. Set your device's date to any point before December 14th, 2023
     - This is so that the profile can be installed despite an expired certificate, and can be set back to normal after the profile is installed.
 1. Open [https://static.itsnebula.net/apple/noota/17.mobileconfig](https://static.itsnebula.net/apple/noota/17.mobileconfig) in your device's browser
-1. Select "Allow" if prompted
+    - If you recieve a prompt that "Your Clock Is Set Incorrectly", click `Show Details` -> `visit this website`. You may then need to reload the page.
+1. Select "Allow", if prompted, followed by "Close"
 1. Open the Settings application
 1. Navigate to `General` -> `Profile`
-    - This may also be called `Profile and Device Management`
-1. Tap on the tvOS Beta Profile you just installed
-1. Tap "Install"
-1. Confirm by tapping "Install" again
+    - This may also be called `Profile and Device Management` or `VPN & Device Management`
+1. Tap `tvOS 17 Beta Software Profile` -> `Install`
     - Enter your passcode if you have one
-1. Tap "Install" for a third time
+1. Confirm by tapping `Install` two more times
```

Comment:
I checked, and the device requests your passcode after tapping `install` once. You then tap `install` 2 more times